test/assert.js

```diff
@@ -2251,26 +2251,31 @@ describe('assert', function () {
         heroes = ['spiderman', 'superman'],
         fn     = function() { obj.value += 5 },
         fnDec  = function() { obj.value -= 20 },
+        getterFn = function() { return obj.value },
         bangFn = function() { obj.str += '!' },
         smFn   = function() { 'foo' + 'bar' },
         batFn  = function() { heroes.push('batman') },
         lenFn  = function() { return heroes.length };
 
     assert.changes(fn, obj, 'value');
+    assert.changes(fn, getterFn, 'value');
     assert.changesBy(fn, obj, 'value', 5);
     assert.changesBy(fn, obj, 'value', -5);
+    // assert.changesBy(fn, getterFn, 'value', 5);
     assert.changesBy(fnDec, obj, 'value', 20);
 
     assert.doesNotChange(smFn, obj, 'value');
+    assert.doesNotChange(smFn, getterFn, 'value');
     assert.changesButNotBy(fnDec, obj, 'value', 1);
+    // assert.changesButNotBy(fnDec, getterFn, 'value', 1);
```

Comment:
Why are these assertions commented out? Do they not work?

Reply:
Yes Cap, it doesn't work. That's why i made #1090 , :smile: